src/main/java/ua/bookstore/online/mapper/BookMapper.java

```diff
@@ -0,0 +1,14 @@
+package ua.bookstore.online.mapper;
+
+import org.mapstruct.Mapper;
+import ua.bookstore.online.config.MapperConfig;
+import ua.bookstore.online.dto.BookDto;
+import ua.bookstore.online.dto.CreateBookRequestDto;
+import ua.bookstore.online.model.Book;
+
+@Mapper(config = MapperConfig.class)
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toModel(CreateBookRequestDto requestDto);
+}
```

Comment:
If you don`t create field Id in your dto, you can receive warm, to avoid it better use annotation  mapping
```suggestion
    @Mapping(target = "id", ignore = true)
    Book toModel(CreateBookRequestDto requestDto);
}
```